content-services/7.0/install/zip/tomcat.md

```diff
@@ -217,6 +217,33 @@ After you've extracted the Content Services distribution zip, several directorie
 | root | `README.txt` | Version information for Content Services and Alfresco Share |
 | | `VERSIONS.md` | List of recommended components for the latest Content Services release |
 
+## Move keystore files to your installation
+
+The new keystore configuration implementation requires it to be configured with
+[JAVA parameters]({% link content-services/7.0/admin/security.md %}#alfresco-keystore-configuration).
+
+1. Move the default keystore files to your installation
+
+    1. Extract from the `alfresco-content-services-distribution-7.0.x.zip` file the following two files:
+
+        * `/keystore/metadata-keystore/keystore`
+        * `/keystore/metadata-keystore/keystore-passwords.properties`
+
+    2. Copy them to your installation at the following location:
+
+        * `<TOMCAT_HOME>/alf_data/keystore/metadata-keystore/keystore`
+        * `<TOMCAT_HOME>/alf_data/keystore/metadata-keystore/keystore-passwords.properties`
+
+2. Configure Tomcat 9 to use the default keystore files
+
+    1. Open `<TOMCAT_HOME>/bin/catalina.bat` in a text editor.
+
+    2. Add the following line to `catalina.bat`:
+
+       `set “JAVA_TOOL_OPTIONS=-Dencryption.keystore.type=JCEKS -Dencryption.cipherAlgorithm=DESede/CBC/PKCS5Padding -Dencryption.keyAlgorithm=DESede -Dencryption.keystore.location=<TOMCAT_HOME>/alf_data/keystore/metadata-keystore/keystore -Dmetadata-keystore.password=mp6yc0UD9e -Dmetadata-keystore.aliases=metadata -Dmetadata-keystore.metadata.password=oKIWzVdEdA -Dmetadata-keystore.metadata.algorithm=DESede”`
```

Comment:
Can we use a codeblock here so it's easier for customers to copy the setting? (also applies to previous version)

Reply:
Done